_posts/2021-04-05-fmf-unified-testing.md

```diff
@@ -128,7 +128,7 @@ The latest testcloud and tmt versions have switched to `qemu:///session` by defa
 
 ## Packit
 
-[Packit](https://packit.dev/) is a tool and a service to automatically package upstream releases into Fedora or COPR.
+[Packit](https://packit.dev/) is a tool and a service to automatically package upstream releases into Fedora or Copr.
```

Comment:
I forgot to mention that I looked up the proper spelling for Copr. It's not an acronym or stylized to be uppercase. It's a proper noun formed as a portmanteau of Community Projects, so the spelling is Copr (and pronunciation is "copper").

Reply:
[indeed](https://copr.fedorainfracloud.org/) -- I need to re-train my muscle memory. Thanks!